utils.py

```diff
@@ -40,3 +42,192 @@ def to_snake(name):
     Convert plugin name from snake case to dash representation
     """
     return name.replace("-", "_")
+
+
+# Changelog to markdown migration
+
+LEGACY_CHANGES_PLACEHOLDER_DATA = """\
+Changes
+*********
+
+Removed due to docs migration process.
+"""
+
+
+def migrate_changelog_to_markdown(plugin_root: str, gh_org: str, gh_repo: str):
+    """
+    - Converts the changelog if there is a CHANGES.rst
+    - Updates the pyproject.toml "tool.towncrier" section to use markdown if it doesnt.
+    """
+    root_path = Path(plugin_root)
+
+    # pyproject update
+    pyproject_file = root_path / "pyproject.toml"
+    if (
+        r'start_string = "[//]: # (towncrier release notes start)\n"'
+        not in pyproject_file.read_text()
+    ):
+        update_pyproject_toml(pyproject_file, gh_org, gh_repo)
+
+    # changelog conversion
+    rst_changes = root_path / "CHANGES.rst"
+    if rst_changes.exists():
+        convert_changelog(rst_changes, gh_org, gh_repo)
+        rst_changes.unlink()
+
+    # manifest update
+    manifest_file = root_path / "MANIFEST.in"
+    if manifest_file.exists():
+        manifest_updated = manifest_file.read_text().replace("CHANGES.rst", "CHANGES.md")
+        manifest_file.write_text(manifest_updated)
+
+    # docs/changes.rst fix
+    legacy_changes_file = root_path / "docs" / "changes.rst"
+    if legacy_changes_file.exists():
+        legacy_changes_file.write_text(LEGACY_CHANGES_PLACEHOLDER_DATA)
```

Comment:
I did this here because some plugins don't have `update_docs` set to true, which could cause some trouble.